httpclient/src/main/kotlin/tbdex/sdk/httpclient/models/CreateExchangeRequest.kt

```diff
@@ -0,0 +1,14 @@
+package tbdex.sdk.httpclient.models
+
+import tbdex.sdk.protocol.models.Rfq
+
+/**
+ * Data class used to type request body to create exchange via TbdexHttpServer.
+ *
+ * @property rfq Rfq tbdex message received.
+ * @property replyTo Optional URL to be included in the request to create exchange.
+ */
+class CreateExchangeRequest(
```

Comment:
Since this is only used in tests, can we move it into `test/` directory? Or is it intended to be used by consumers of this library in some way?

Reply:
we do use this in Tbdexhttpclient#sendMessage method where i'm passing in `rfq` and `replyTo` (the new overloaded method)

```kotlin
val body: RequestBody = Json.stringify(CreateExchangeRequest(message, replyTo))
      .toRequestBody(jsonMediaType)
```